internal/setup/setup.go

```diff
@@ -44,6 +45,10 @@ type Package struct {
 	Slices  map[string]*Slice
 }
 
+func (p *Package) MarshalYAML() (interface{}, error) {
```

Comment:
Please also add the assertions that the type implements the Marshaler inteface like what we discussed in the other comment.